### PR TITLE
added basic ownCloud setup manual

### PR DESCRIPTION
**Pull request by spacecodeit**
First try to contribute to the lab.

Please note:
There is no ownCloud logo with transparent background and it is not allowed to invert the image: https://owncloud.org/trademark/

### CODE REVIEW

File: source/guide_owncloud.rst [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+.. sidebar:: Logo
+  
+  .. image:: _static/images/owncloud.png 
+      :align: center
+
+#########
+ownCloud
+#########
+
+ownCloud_ is an open source cloud solution written in PHP and distributed unter the GAPLv3 licence.
```
Comment (SalocinHB) [inline source/guide_owncloud.rst]: The license is called AGPLv3

File: source/guide_owncloud.rst [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+.. include:: includes/my-print-defaults.rst
+
+If you want to use your cloud with your own domain you need to setup your domain first:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+``cd`` to your `document root`_, then download and the latest release of the ownCloud and extract it:
```
Comment (SalocinHB) [inline source/guide_owncloud.rst]: Please tell users where to find the link to the latest release. Lose the 'and' after download.

File: source/guide_owncloud.rst [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+.. warning:: Replace ``<username>`` with your username!
+
+::
+
+ *  *  *  *  * /usr/bin/php71 -f var/www/virtual/<username>/html/cron.php
+
+Updates
+=======
+
+There are many different ways to update your ownCloud installation.
```
Comment (SalocinHB) [inline source/guide_owncloud.rst]: Please remove this line if you're not going to explain the other options.

File: source/guide_owncloud.rst [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+::
+
+ *  *  *  *  * /usr/bin/php71 -f var/www/virtual/<username>/html/cron.php
+
+Updates
+=======
+
+There are many different ways to update your ownCloud installation. 
+
+The easiest one is to use the web updater provided in the admin section of the online panel.
```
Comment (SalocinHB) [inline source/guide_owncloud.rst]: Is it really called 'online panel'?

File: source/guide_owncloud.rst [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+To get the best performance ownCloud suggests to add a local cronjob.
+
+Add the following cronjob to your cronjobs_ configuration:
+
+.. warning:: Replace ``<username>`` with your username!
+
+::
+
+ *  *  *  *  * /usr/bin/php71 -f var/www/virtual/<username>/html/cron.php
```
Comment (SalocinHB) [inline source/guide_owncloud.rst]: The path to the script is missing a `/` in the beginning. Please don't link to the php binary directly, use our wrapper `/usr/bin/php`.

File: source/guide_owncloud.rst [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+  * your MySQL hostname, username and password: the hostname is ``localhost`` and you should know your MySQL credentials_ by now. If you don't, start reading again at the top.
+  * your ownCloud database name: we suggest you use an additional_ database. For example: isabell_owncloud
+
+Tuning
+======
+
+To get the best performance ownCloud suggests to add a local cronjob.
+
+Add the following cronjob to your cronjobs_ configuration:
```
Comment (SalocinHB) [inline source/guide_owncloud.rst]: 'Add the following cronjob to your `crontab`_:'

File: source/guide_owncloud.rst [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+.. warning:: Replace ``<username>`` with your username!
+
+::
+
+ *  *  *  *  * /usr/bin/php71 -f var/www/virtual/<username>/html/cron.php
+
```
Comment (SalocinHB) [inline source/guide_owncloud.rst]: Memcaching (APCu) seems worth a mention.

Comment (spacecodeit) [reply source/guide_owncloud.rst]: When I try to add APCu to my custom php.ini file I get the following error:
PHP Warning:  Module 'apcu' already loaded in Unknown on line 0
But the ownCloud web interface says that no cache is available.

Which steps are needed to enable APCu? I can't find anything about that in the manual.

Comment (SalocinHB) [reply source/guide_owncloud.rst]: It is already enabled in PHP, you need to enable it in ownCloud. Check their manual.

**Comment by spacecodeit** [comment]
I have changed all mentioned stuff and added how to enable Memcaching.

**Pull request closed without merging**